<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Premier</class>
 <widget class="QDialog" name="Premier">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>938</width>
    <height>544</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>350</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="3" rowspan="2">
    <widget class="QPushButton" name="pushButton">
     <property name="text">
      <string>?</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLineEdit" name="lineEdit"/>
       </item>
       <item>
        <widget class="QPushButton" name="seatchB">
         <property name="text">
          <string>Поиск</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QListWidget" name="listWidget"/>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_2">
         <property name="text">
          <string>Удалить выполненные</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="addExercise">
         <property name="text">
          <string>Добавить</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QListWidget" name="listWidget_2"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QPushButton" name="pushButton_5">
         <property name="text">
          <string>Удалить выполненные</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="addFood">
         <property name="text">
          <string>Добавить</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item row="2" column="2" colspan="2">
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QCalendarWidget" name="calendarWidget">
       <property name="minimumSize">
        <size>
         <width>350</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>600</width>
         <height>600</height>
        </size>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="0" column="0" rowspan="2">
    <widget class="QLabel" name="zag">
     <property name="text">
      <string>Fitness Planner</string>
     </property>
    </widget>
   </item>
   <item row="0" column="2">
    <widget class="QTimeEdit" name="timeEdit"/>
   </item>
   <item row="1" column="2">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
